amethyst_tiles/src/map.rs

```diff
@@ -101,7 +101,6 @@ pub trait MapStorage<T: Tile> {
 
 /// Concrete implementation of a generic 3D `TileMap` component. Accepts a `Tile` type and `CoordinateEncoder` type,
 /// creating a flat 1D array storage which is spatially partitioned utilizing the provided encoding scheme.
-/// creating a flat 1D array storage which is spatially partitioned utilizing the provided encoding scheme.
 ///
 /// The defualt encoding scheme is `MortonEncoder2D`, which allows for arbitrary X, Y and Z coordinate sizes while
```

Comment:
You could also change defualt to default and fix that typo (?)

Reply:
Good catch. 😄